packages/odyssey-react-mui/src/labs/Grid.tsx

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import styled from "@emotion/styled";
+import { Children, ReactNode, memo } from "react";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "../OdysseyDesignTokensContext";
+
+type SupportedPaneRatios =
+  | [1]
+  | [1, 1]
+  | [1, 2]
+  | [2, 1]
+  | [1, 3]
+  | [3, 1]
+  | [1, 1, 1]
+  | [1, 2, 1]
+  | [2, 1, 1]
+  | [1, 1, 2]
+  | [1, 1, 1, 1];
```

Comment:
Open to narrowing this type to support less ratios OOTB so any new ratios can be additive

Reply:
I don't think we need these
```js
[1, 2, 1]
[2, 1, 1]
[1, 1, 2]
```